apps/researcher/src/app/[locale]/objects/[id]/(provenance)/categorize-timeline-events.ts

```diff
@@ -0,0 +1,73 @@
+import {LabeledProvenanceEvent, TimeLineEvent} from './definitions';
+
+interface CategorizedEvents {
+  rangeEvents: TimeLineEvent[];
+  singleEvents: TimeLineEvent[];
+  eventsWithoutDates: LabeledProvenanceEvent[];
+}
+
+// Events within one event group are rendered as a single timeline event,
```

Comment:
Very minor: instead of putting a `//` at the start of each line, use a multi-line comment: `/* ... */`?